### PR TITLE
Improve subscription-matcher Messages description

### PR DESCRIPTION
**Pull request by mackdk**
# Description

Expand documentation about subscription matching messages 

# Target branches

Backport targets:

- master
- 4.3 -- https://github.com/uyuni-project/uyuni-docs/pull/2746

# Links
- This PR tracks issue https://github.com/SUSE/spacewalk/issues/22893

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Added details information about the messages produced by subscription matcher
```
Comment (keichwa) [inline CHANGELOG.md:1]:
```suggestion
- Added detailed information about the messages produced by subscription matcher 
```

File: modules/administration/pages/subscription-matching.adoc [hunk 2]
```diff
@@ -16,7 +16,7 @@ The report includes product names and the number of systems that remain unmatche
 The [guimenu]``Pins`` tab allows you to associate individual clients to the relevant subscription.
 This is especially useful if the subscription manager is not automatically associating clients to subscriptions successfully.
 
-The [guimenu]``Messages`` tab shows any errors that occurred during the matching process.
+The [guimenu]``Messages`` tab shows all the messages generated by subscription matcher during the matching process. They provide information to help understand the results and to improve the matching.
```
Comment (0rnela) [inline modules/administration/pages/subscription-matching.adoc:19]:
```suggestion
The [guimenu]``Messages`` tab shows all the messages generated by subscription matcher during the matching process. 
They provide information to help understand the results and to improve the matching.
```

**Review by 0rnela — APPROVED** [review]

File: modules/reference/pages/audit/audit-subscription-matching.adoc [hunk 3]
```diff
@@ -27,15 +27,19 @@ Table entries that have already expired are shown in grayscale.
...
+| Two subscriptions with the same part number are in a bundle | Informational | Two subscriptions were identified to be part of a bundle and, as result, they were merged together during the computation process. | No actions needed.
+| Two subscriptions with the same part number (and other properties) have been merged together | Informational | Two subscription shared multiple common properties and they were merged together during the computation process. | No actions needed.
+| Pinned match adjusted due to merged subscriptions | Informational | A user defined pin was updated since the chosen subscription was merged with another one. | No actions needed.
+| Subscription with unsupported part number and no associated product has been ignored. | Informational | An unknown subscription without any associated product has been detected and it was ignored during the mathing process. | No actions needed.
```
Comment (keichwa) [inline modules/reference/pages/audit/audit-subscription-matching.adoc:42]:
```suggestion
| Subscription with unsupported part number and no associated product has been ignored. | Informational | An unknown subscription without any associated product has been detected and it was ignored during the matching process. | No actions needed.
```

**Review by keichwa — APPROVED** [review]
just cosmetic

**Pull request merged**